lib/ws-kernel-picker.coffee

```diff
@@ -0,0 +1,117 @@
+{SelectListView} = require 'atom-space-pen-views'
+_ = require 'lodash'
+
+Config = require './config'
+services = require './jupyter-js-services-shim'
+WSKernel = require './ws-kernel'
+uuid = require 'uuid'
+
+class CustomListView extends SelectListView
+    initialize: (@emptyMessage, @onConfirmed) ->
+        super
+        @addClass('watch-language-picker')
+        @storeFocusedElement()
```

Comment:
Thanks for pointing this out.

I'm not sufficiently familiar with atom to know a fix offhand, so I've made a note to look into this in the future (i.e. another PR).